src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageTransportBindingElement.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Net;
+using CoreWCF.Configuration;
+using CoreWCF.Queue.Common;
+using CoreWCF.Queue.Common.Configuration;
+using Azure.Storage.Queues;
+
+namespace CoreWCF.Channels
+{
+    public class AzureQueueStorageTransportBindingElement : QueueBaseTransportBindingElement
+    {
+        /// <summary>
+        /// Creates a new instance of the AzureQueueStorageTransportBindingElement Class using the default protocol.
+        /// </summary>
+        public AzureQueueStorageTransportBindingElement()
+        {
+        }
+
+        public override BindingElement Clone()
+        {
+            return new AzureQueueStorageTransportBindingElement();
+        }
+
+
+        public override T GetProperty<T>(BindingContext context)
+        {
+            if (context == null)
+            {
+                throw DiagnosticUtility.ExceptionUtility.ThrowHelperArgumentNull(nameof(context));
+            }
+
+            if (typeof(T) == typeof(ISecurityCapabilities))
+            {
```

Comment:
The base GetProperty will return null for any type not recognized so you don't need to do this. You probably could get rid of this method, but I think it would be good to keep even if all it does is call base.GetProperty as it's convention to override this method so it would act as a place holder if something were to be put here in the future.

Reply:
ok. removing this check.